photos.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'>
+    <meta name='description'>
+    <title>Sedona Photos</title>
+    <link rel="stylesheet" href="">
+  </head>
+
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <nav class="main-nav">
+          <ul class="main-nav__list">
+            <li class="main-nav__item"><a href="index.html">Home</a></li>
+            <li class="main-nav__item main-nav__item--active"><a href="photos.html">Photos and videos</a></li>
+            <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+
+      <main class="main-content">
+        <section class="preamble">
+          <div class="preamble__inner">
+            <h2 class="preamble__header">Photo and video</h2>
+            <p class="preamble__paragraph">Can't decide to travel because of the right time? We hope photos will help you forget about politics and economics.</p>
+          </div>
+        </section>
+
+        <section class="gallery__main-photo">
```

Comment:
Where's is `gallery` block? Every image element inside of the block has to be interchangeable. So having separate elements for every image with a different class is not very flexible, and adds a lot of boilerplate code. Images has be it's own blocks, with same classes when an image is either put just as `img`, or a modifier class that changes the background.

Reply:
As I've noticed the images on the PSD file do not Change the size as a responsive images. the image is cropped with the borders. Therefore I thought it will be another place where the picture will be used as a background instead of the <img> tag.

Reply:
Yes, that's fine then as I said above :
> or a modifier class that changes the background.

but you're still missing `gallery` block, you are using element right away `gallery__main-photo`